guides/maintainer/conflict-resolution.md

```diff
@@ -0,0 +1,29 @@
+# Resolving technical conflicts within a SIG
```

Comment:
should this be linked to something? I was expecting this in the README?

Reply:
going to merge this now, raise a follow up issue for further discussion